src/julia-parser.scm

```diff
@@ -549,7 +549,9 @@
                            op))
                         (else '|.|)))))
 
-          ((opchar? c)  (read-operator port (read-char port)))
+          ((if (opchar? c)
+              (accum-julia-symbol c port)
```

Comment:
No, this is wrong; `accum-julia-symbol` reads an identifier (not an operator) from the input stream, leaving nothing else for `read-operator` to read.

I think you will need to export a new flisp function to normalize a symbol

Reply:
I added a new commit. It works now after `make cleanall` and `make`